src/bundled.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import CioQuiz from './components/CioQuiz';
+import './styles.css';
+
+const ConstructorioQuizzes = ({
+  selector,
+  apiKey,
+  quizId,
+  cioJsClient,
+  resultsPageOptions,
+  quizVersionId,
+  includeCSS = true,
+}) => {
+  if (document) {
+    if (!includeCSS) {
+      const stylesheet = document.getElementById('cio-quizzes-styles');
+
+      if (stylesheet) {
+        stylesheet.disabled = true;
+      }
+    }
```

Comment:
Forgot to mention, but if we're doing it this way, should we also have an else statement here with:

`document.getElementById('cio-quizzes-styles').disabled = false`